pkg/genericapiserver/config.go

```diff
@@ -440,3 +443,10 @@ func DefaultAndValidateRunOptions(options *options.ServerRunOptions) {
 		}
 	}
 }
+
+func NewRequestInfoResolver(c *Config) *request.RequestInfoResolver {
+	return &request.RequestInfoResolver{
+		APIPrefixes:          sets.NewString(strings.Trim(c.APIPrefix, "/"), strings.Trim(c.APIGroupPrefix, "/")), // all possible API prefixes
+		GrouplessAPIPrefixes: sets.NewString(strings.Trim(c.APIPrefix, "/")),                                      // APIPrefixes that won't have groups (legacy)
```

Comment:
make sure c.APIPrefix and c.APIGroupPrefix are not empty strings before including them in the sets. When we eventually start up a generic API server that is only serving grouped APIs, I *think* I would expect `APIPrefix` to be empty